examples/s2i-python-sample/simpleTest.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+#
+# Meta test family (MTF) is a tool to test components of a modular Fedora:
+# https://docs.pagure.org/modularity/
+# Copyright (C) 2017 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# he Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Authors: Jan Scotka <jscotka@redhat.com>
+#
+
+from moduleframework import module_framework
+import time
+import urllib
+
+
+class simpleTests(module_framework.ContainerAvocadoTest):
+    """
+    :avocado: enable
+    """
+    ipaddr = module_framework.trans_dict["GUESTIPADDR"]
+    port = "8080"
+
+    def life_check(self, ipaddr=None, port=None, textcheck="Hello from gunicorn WSGI application!"):
+        ipaddr = ipaddr or self.ipaddr
+        port = port or self.port
+        # wait until container initialized (without probe)
+        time.sleep(3)
+        urlfd = urllib.urlopen("http://%s:%s" % (ipaddr, port))
+        self.assertIn(textcheck, urlfd.read())
+        urlfd.close()
+
+    def test_via_curl(self):
+        self.start()
+        time.sleep(3)
+        output = self.runHost("curl http://%s:%s" % (self.ipaddr, self.port))
+        self.assertIn("Hello from gunicorn WSGI application!", output.stdout)
+
+    def test_basic(self):
+        self.start()
+        self.life_check()
+
+    def test_another_port(self):
+        self.backend.info["start"]="docker run -p 9999:8080"
+        self.start()
+        self.life_check(port="9999")
```

Comment:
one more test could be to check `s2i usage` whether it's present

Reply:
test added as separate class in this file